web/docs/database/crud.md

```diff
@@ -1,3 +1,746 @@
 ---
 title: Automatic CRUD
 ---
+
+import { Required } from '@site/src/components/Required';
+import { ShowForTs } from '@site/src/components/TsJsHelpers';
+
+Writing similar [Queries and Actions](/docs/database/operations) sometimes gets tiresome, especially if they are really simple. This is where Automatic CRUD really shines!
```

Comment:
I would motivate it differently:

> If you wrote a few full-stack apps, you probably noticed that some things tend to repeat: listing some data, creating new data, editing it and deleting it. We wanted to make it easier to handle these boring bits by offering a higher level concept Automatic CRUD.

Something along those lines, change it to what sounds good to you.